auslib/log.py

```diff
@@ -53,28 +52,43 @@ class JsonLogFormatter(logging.Formatter):
     Adapted from:
     https://github.com/mozilla-services/mozservices/blob/master/mozsvc/util.py#L106
     """
+
     LOGGING_FORMAT_VERSION = "2.0"
 
     # Map from Python logging to Syslog severity levels
-    SYSLOG_LEVEL_MAP = {
-        50: 2,  # CRITICAL
-        40: 3,  # ERROR
-        30: 4,  # WARNING
-        20: 6,  # INFO
-        10: 7,  # DEBUG
-    }
+    SYSLOG_LEVEL_MAP = {50: 2, 40: 3, 30: 4, 20: 6, 10: 7}  # CRITICAL  # ERROR  # WARNING  # INFO  # DEBUG
```

Comment:
this is weird

Reply:
The Agent is a separate app in the repo; it doesn't share any code with Balrog. (Does that make Balrog a monorepo? 🤣)